apps/arkmarket/src/types/index.ts

```diff
@@ -120,11 +120,11 @@ export interface PortfolioToken {
 
 export interface TokenOffer {
   expire_at: number;
-  floor_difference: number | null;
+  floor_difference?: number;
   hash: string;
   offer_id: number;
   price: string;
-  source: string | null;
```

Comment:
I previously added `null` because sometimes `source` was `null` from the API, which was breaking the page, are we sure that `source` can't be `null` now?